### ISSUE TITLE
Enlarge the “What Happens Next” font size

### ISSUE THREAD

**Issue by oliver2g** [cityofaustin/techstack#1601]
The OPO Advisory Committee is concerned that the “What Happens Next” link is not visible enough on the confirmation page. While user research shows that the link is getting seen (we know because they click on it), some users have said that that information would also be useful at the beginning of the flow. 
![complaint confirmation issue-03-26 at 2.45.45 PM.png](https://images.zenhubusercontent.com/5a9edc4e4b5806bc2bc9a740/2db151df-b286-48a1-8e0b-950b3b95f005)

**Comment by sergiogcx** [comment]
I can work on this today, but it would be nice to have a specification on the size:

This is it on *2.0 rem* (about 16 to 20 pixels (depending on the device):
![Screen Shot 2019-04-01 at 3.49.54 PM.png](https://images.zenhubusercontent.com/5b7edad7290aac725aec290c/f933b421-c063-4bb2-bf9f-44ecab5b8b23)

This is the link on *1.5 rem* (about 14-15 pixels), bold:

![Screen Shot 2019-04-01 at 3.50.19 PM.png](https://images.zenhubusercontent.com/5b7edad7290aac725aec290c/e0553749-f67e-4194-8885-d5a14a6c1e26)

**Comment by oliver2g** [comment]
good question @sergiogcx. I would say make the font similar to the confirmation number. @toribr and @chasechenevert any thoughts on this from a ux perspective?

**Comment by sergiogcx** [comment]
The confirmation has size: 1.8 rem, bold: 700. If we make them the same, this is what it looks like:

![Screen Shot 2019-04-01 at 3.56.44 PM.png](https://images.zenhubusercontent.com/5b7edad7290aac725aec290c/fdabddb2-8b0b-4982-abd2-d6d3285b47c4)

**Comment by toribr** [comment]
Please wait on implementing anything for this issue, until UX has a chance to correctly research.

**Comment by sergiogcx** [comment]
Ok, I will put this back on deck.

**Comment by oliver2g** [comment]
I'll push this "why" question back to Farah and the OPO advisory committee

**Comment by oliver2g** [comment]
@toribr will you have any UX recommendations on this issue for this Sprint?

**Comment by toribr** [comment]
In order to address the "spirit" of this issue, changes need to be made in two places:
1) The landing page, which needs an additional sentence. (Note: this will need a Spanish translation)
![1 2 - Landing – 2](https://user-images.githubusercontent.com/3790067/57483649-ea08f200-726c-11e9-8315-b7d0b137e9a9.png)
2) The confirmation page: make existing text "the process" a link as shown. Also, remove the progress bar from this page (the process is over). This will help move more content up into the viewport.
![8 - Success – 1](https://user-images.githubusercontent.com/3790067/57483750-1d4b8100-726d-11e9-83c4-21a846277ffc.png)

**Comment by toribr** [comment]
@oliver2g Design is done, I'll leave it to you to move this to where you next want it in the pipeline.

**Comment by oliver2g** [comment]
I will review this design with Sara Peralta before assigning it to dev

**Comment by oliver2g** [comment]
@sergiogcx or @Niicck this is an issue that needs to be implemented on the English and Spanish complaint form in this sprint. It's a request directly from Farah. Does either of you have bandwidth?

@desigonz can you translate the sentence from Tori's mock above: "Learn about the complaint investigation process." I don't want to take the simplicity of the sentence for granted, so let me know if this is a stretch.

**Comment by desigonz** [comment]
Use "Aprenda sobre el proceso de investigación de quejas" as a placeholder—but I'm going to throw this onto the list for official translations.

**Comment by Niicck** [comment]
@oliver2g @sergiogcx @toribr 
Lo and Behold comrades:

We got a new link on the intro page.
https://opo.austintexas.io/policia-queja/1601-learn-more/introduccion
<img width="1174" alt="Screen Shot 2019-05-18 at 2 28 13 PM" src="https://user-images.githubusercontent.com/14187277/57974147-45c43100-7979-11e9-8144-4e9fa4d10472.png">

We linked up "process" on the confirmation page and took out the progress bar.
https://opo.austintexas.io/policia-queja/1601-learn-more/confirmation
<img width="1164" alt="Screen Shot 2019-05-18 at 2 33 27 PM" src="https://user-images.githubusercontent.com/14187277/57974186-02b68d80-797a-11e9-9a08-b0f81d1127c4.png">

The noticeability of "What Happens Next" still seems kinda tiny on the confirmation page. But if Tori and Sara are happy, then we can push this into prod.
